specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Builds.json

```diff
@@ -388,6 +443,36 @@
         "server"
       ]
     },
+    "BuildToken": {
+      "description": "Build Auth Token.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
```

Comment:
>ProxyResource [](http://example.com/codeflow?start=94&length=13)

Now that this is POST you dont need to model this as a ARM resource. This will confuse users who start seeing a resource id here.

Reply:
I updated this to be consistent with the rest of the Microsoft.App POST actions, such as [`containerapp/getAuthToken`](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/ContainerApps.json#L1335-L1364)